app/plot_app/configured_plots.py

```diff
@@ -186,19 +193,97 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
 
     if data_plot.finalize() is not None: plots.append(data_plot)
 
-
+    # VTOL tailistter orientation conversion, if relevant
+    if is_vtol_tailsitter:
+        print(vtol_states)
+        for d in data:
+            if d.name == 'vehicle_attitude':
```

Comment:
If the log does not contain either of these topics there's undefined variables below. Make sure that is handled gracefully.